Examples/PublisherExampleSwiftUI/PublisherExampleSwiftUI/Model/Uploads/UploadRequest.swift

```diff
@@ -1,8 +1,19 @@
 import Foundation
 import AblyAssetTrackingPublisher
 
-struct UploadRequest {
-    var data: LocationHistoryData
+struct UploadRequest: Codable {
+    enum `Type`: CustomStringConvertible, Codable {
```

Comment:
Not a biggest fan of using the metatype keyword as a name for the enum. Maybe we could name it `UploadType` or something along those lines instead?

Reply:
Sure, changed 🙂